README.md

```diff
@@ -519,7 +511,7 @@ The following fields are allowed in the alias structure:
     * `alias.id.write` contains the ID of the object which will be set when alias is written
     * `alias.id.read` contains the ID of the object which will be mirrored to the alias object/state   
 * `alias.read` can optionally contain a read script (will be evaluated) to calculate the alias value when the target state changes
-* `alias.write` can optionally contain a write script (will be evaluated) to calculate the target value if the alias value is changed
+* `alias.write` can optionally contain a write-script (will be evaluated) to calculate the target value if the alias value is changed
```

Comment:
read script stays the same and write-script gets a hypen, does not make sense, would keep it like before